[libfreenect2] Build error on x64-osx

Package: libfreenect2[core,opengl]:x64-osx -> 0.2.0#7
**Host Environment**
- Host: arm64-osx
- Compiler: AppleClang 13.0.0.13000029
-    vcpkg-tool version: 2999-12-31-unknownhash-debug
    vcpkg-scripts version: 4fff1e9cf 2022-01-17 (17 hours ago)


**To Reproduce**
`vcpkg install `


**Failure logs**
```
-- Using cached OpenKinect-libfreenect2-v0.2.0.tar.gz.
-- Extracting source /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/downloads/OpenKinect-libfreenect2-v0.2.0.tar.gz
-- Applying patch fix-dependency-libusb.patch
-- Using source at /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean
CMake Deprecation Warning at scripts/cmake/vcpkg_check_features.cmake:182 (message):
  calling `vcpkg_check_features` without the `FEATURES` keyword has been
  deprecated.

      Please add the `FEATURES` keyword to the call.
Call Stack (most recent call first):
  ports/libfreenect2/portfile.cmake:20 (vcpkg_check_features)
  scripts/ports.cmake:145 (include)


-- Found external ninja('1.10.2').
-- Configuring x64-osx-dbg
-- Configuring x64-osx-rel
-- Building x64-osx-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:157 (message):
    Command failed: /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/downloads/tools/cmake-3.21.1-osx/cmake-3.21.1-macos-universal/CMake.app/Contents/bin/cmake --build . --config Debug --target install -- -v -j11
    Working Directory: /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/x64-osx-dbg
    See logs for more information:
      /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/install-x64-osx-dbg-out.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_cmake.cmake:106 (vcpkg_execute_build_process)
  scripts/cmake/vcpkg_install_cmake.cmake:44 (vcpkg_build_cmake)
  ports/libfreenect2/portfile.cmake:34 (vcpkg_install_cmake)
  scripts/ports.cmake:145 (include)



```
<details><summary>/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/install-x64-osx-dbg-out.log</summary>

```
[1/26] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DRESOURCES_INC -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/include/internal -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/x64-osx-dbg -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/src/tinythread -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/libusb-1.0 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include -fPIC -Wall -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -MD -MT CMakeFiles/generate_resources_tool.dir/tools/generate_resources.cpp.o -MF CMakeFiles/generate_resources_tool.dir/tools/generate_resources.cpp.o.d -o CMakeFiles/generate_resources_tool.dir/tools/generate_resources.cpp.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/tools/generate_resources.cpp
[2/26] : && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -Wall -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names  CMakeFiles/generate_resources_tool.dir/tools/generate_resources.cpp.o -o bin/generate_resources_tool   && :
[3/26] cd /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean && /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/x64-osx-dbg/bin/generate_resources_tool /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean > /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/x64-osx-dbg/resources.inc.h
[4/26] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DRESOURCES_INC -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/include/internal -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/x64-osx-dbg -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/src/tinythread -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/libusb-1.0 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include -fPIC -Wall -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -MD -MT CMakeFiles/freenect2.dir/src/event_loop.cpp.o -MF CMakeFiles/freenect2.dir/src/event_loop.cpp.o.d -o CMakeFiles/freenect2.dir/src/event_loop.cpp.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/src/event_loop.cpp
...
Skipped 13 lines
...

[18/26] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DRESOURCES_INC -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/include/internal -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/x64-osx-dbg -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/src/tinythread -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/libusb-1.0 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include -fPIC -Wall -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -MD -MT CMakeFiles/freenect2.dir/src/tinythread/tinythread.cpp.o -MF CMakeFiles/freenect2.dir/src/tinythread/tinythread.cpp.o.d -o CMakeFiles/freenect2.dir/src/tinythread/tinythread.cpp.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/src/tinythread/tinythread.cpp
[19/26] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DRESOURCES_INC -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/include/internal -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/x64-osx-dbg -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/src/tinythread -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/libusb-1.0 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include -fPIC -Wall -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -MD -MT CMakeFiles/freenect2.dir/src/command_transaction.cpp.o -MF CMakeFiles/freenect2.dir/src/command_transaction.cpp.o.d -o CMakeFiles/freenect2.dir/src/command_transaction.cpp.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/src/command_transaction.cpp
[20/26] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DRESOURCES_INC -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/include/internal -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/x64-osx-dbg -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/src/tinythread -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/libusb-1.0 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include -fPIC -Wall -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -MD -MT CMakeFiles/freenect2.dir/src/turbo_jpeg_rgb_packet_processor.cpp.o -MF CMakeFiles/freenect2.dir/src/turbo_jpeg_rgb_packet_processor.cpp.o.d -o CMakeFiles/freenect2.dir/src/turbo_jpeg_rgb_packet_processor.cpp.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/src/turbo_jpeg_rgb_packet_processor.cpp
[21/26] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DRESOURCES_INC -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/include/internal -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/x64-osx-dbg -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/src/tinythread -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/libusb-1.0 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include -fPIC -Wall -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -MD -MT examples/CMakeFiles/Protonect.dir/Protonect.cpp.o -MF examples/CMakeFiles/Protonect.dir/Protonect.cpp.o.d -o examples/CMakeFiles/Protonect.dir/Protonect.cpp.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/examples/Protonect.cpp
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/examples/Protonect.cpp:339:26: warning: unused variable 'ir' [-Wunused-variable]
    libfreenect2::Frame *ir = frames[libfreenect2::Frame::Ir];
                         ^
1 warning generated.
[22/26] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DRESOURCES_INC -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/include/internal -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/x64-osx-dbg -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/src/tinythread -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/libusb-1.0 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include -fPIC -Wall -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -MD -MT CMakeFiles/freenect2.dir/src/libfreenect2.cpp.o -MF CMakeFiles/freenect2.dir/src/libfreenect2.cpp.o.d -o CMakeFiles/freenect2.dir/src/libfreenect2.cpp.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/src/libfreenect2.cpp
[23/26] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DRESOURCES_INC -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/include/internal -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/x64-osx-dbg -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/src/tinythread -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/libusb-1.0 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include -fPIC -Wall -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -MD -MT CMakeFiles/freenect2.dir/src/vt_rgb_packet_processor.cpp.o -MF CMakeFiles/freenect2.dir/src/vt_rgb_packet_processor.cpp.o.d -o CMakeFiles/freenect2.dir/src/vt_rgb_packet_processor.cpp.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/libfreenect2/src/v0.2.0-69ac2224d9.clean/src/vt_rgb_packet_processor.cpp
[24/26] : && /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/downloads/tools/cmake-3.21.1-osx/cmake-3.21.1-macos-universal/CMake.app/Contents/bin/cmake -E rm -f lib/libfreenect2.a && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ar qc lib/libfreenect2.a  CMakeFiles/freenect2.dir/src/transfer_pool.cpp.o CMakeFiles/freenect2.dir/src/event_loop.cpp.o CMakeFiles/freenect2.dir/src/usb_control.cpp.o CMakeFiles/freenect2.dir/src/allocator.cpp.o CMakeFiles/freenect2.dir/src/frame_listener_impl.cpp.o CMakeFiles/freenect2.dir/src/packet_pipeline.cpp.o CMakeFiles/freenect2.dir/src/rgb_packet_stream_parser.cpp.o CMakeFiles/freenect2.dir/src/rgb_packet_processor.cpp.o CMakeFiles/freenect2.dir/src/depth_packet_stream_parser.cpp.o CMakeFiles/freenect2.dir/src/depth_packet_processor.cpp.o CMakeFiles/freenect2.dir/src/cpu_depth_packet_processor.cpp.o CMakeFiles/freenect2.dir/src/resource.cpp.o CMakeFiles/freenect2.dir/src/command_transaction.cpp.o CMakeFiles/freenect2.dir/src/registration.cpp.o CMakeFiles/freenect2.dir/src/logging.cpp.o CMakeFiles/freenect2.dir/src/libfreenect2.cpp.o CMakeFiles/freenect2.dir/src/tinythread/tinythread.cpp.o CMakeFiles/freenect2.dir/src/vt_rgb_packet_processor.cpp.o CMakeFiles/freenect2.dir/src/turbo_jpeg_rgb_packet_processor.cpp.o && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib lib/libfreenect2.a && /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/downloads/tools/cmake-3.21.1-osx/cmake-3.21.1-macos-universal/CMake.app/Contents/bin/cmake -E touch lib/libfreenect2.a && :
[25/26] : && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -Wall -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names  examples/CMakeFiles/Protonect.dir/Protonect.cpp.o -o bin/Protonect  lib/libfreenect2.a  /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/debug/lib/libusb-1.0.a  -lpthread  -framework  VideoToolbox  -framework  CoreFoundation  -framework  CoreMedia  -framework  CoreVideo  /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/debug/lib/libturbojpeg.a && :
FAILED: bin/Protonect 
: && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -Wall -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names  examples/CMakeFiles/Protonect.dir/Protonect.cpp.o -o bin/Protonect  lib/libfreenect2.a  /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/debug/lib/libusb-1.0.a  -lpthread  -framework  VideoToolbox  -framework  CoreFoundation  -framework  CoreMedia  -framework  CoreVideo  /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/debug/lib/libturbojpeg.a && :
Undefined symbols for architecture x86_64:
  "_IOCreatePlugInInterfaceForService", referenced from:
      _darwin_claim_interface in libusb-1.0.a(darwin_usb.o)
      _darwin_device_from_service in libusb-1.0.a(darwin_usb.o)
  "_IOIteratorNext", referenced from:
      _darwin_scan_devices in libusb-1.0.a(darwin_usb.o)
      _darwin_check_configuration in libusb-1.0.a(darwin_usb.o)
      _darwin_devices_detached in libusb-1.0.a(darwin_usb.o)
      _darwin_devices_attached in libusb-1.0.a(darwin_usb.o)
      _darwin_clear_iterator in libusb-1.0.a(darwin_usb.o)
      _darwin_get_interface in libusb-1.0.a(darwin_usb.o)
  "_IOKitWaitQuiet", referenced from:
      _darwin_hotplug_poll in libusb-1.0.a(darwin_usb.o)
  "_IONotificationPortCreate", referenced from:
      _darwin_event_thread_main in libusb-1.0.a(darwin_usb.o)
  "_IONotificationPortDestroy", referenced from:
      _darwin_event_thread_main in libusb-1.0.a(darwin_usb.o)
  "_IONotificationPortGetRunLoopSource", referenced from:
      _darwin_event_thread_main in libusb-1.0.a(darwin_usb.o)
  "_IOObjectRelease", referenced from:
      _darwin_claim_interface in libusb-1.0.a(darwin_usb.o)
      _darwin_kernel_driver_active in libusb-1.0.a(darwin_usb.o)
      _darwin_scan_devices in libusb-1.0.a(darwin_usb.o)
      _darwin_event_thread_main in libusb-1.0.a(darwin_usb.o)
      _get_device_port in libusb-1.0.a(darwin_usb.o)
      _darwin_check_configuration in libusb-1.0.a(darwin_usb.o)
      _darwin_devices_detached in libusb-1.0.a(darwin_usb.o)
      ...
  "_IORegistryEntryCreateCFProperty", referenced from:
      _darwin_kernel_driver_active in libusb-1.0.a(darwin_usb.o)
      _get_ioregistry_value_number in libusb-1.0.a(darwin_usb.o)
      _get_ioregistry_value_data in libusb-1.0.a(darwin_usb.o)
  "_IORegistryEntryGetParentEntry", referenced from:
      _get_device_port in libusb-1.0.a(darwin_usb.o)
      _get_device_parent_sessionID in libusb-1.0.a(darwin_usb.o)
  "_IOServiceAddMatchingNotification", referenced from:
      _darwin_event_thread_main in libusb-1.0.a(darwin_usb.o)
  "_IOServiceGetMatchingServices", referenced from:
      _usb_setup_device_iterator in libusb-1.0.a(darwin_usb.o)
  "_IOServiceMatching", referenced from:
      _darwin_event_thread_main in libusb-1.0.a(darwin_usb.o)
      _usb_setup_device_iterator in libusb-1.0.a(darwin_usb.o)
  "_kIOMasterPortDefault", referenced from:
      _darwin_hotplug_poll in libusb-1.0.a(darwin_usb.o)
      _darwin_event_thread_main in libusb-1.0.a(darwin_usb.o)
      _usb_setup_device_iterator in libusb-1.0.a(darwin_usb.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
ninja: build stopped: subcommand failed.
```
</details>

**Additional context**
<details><summary>vcpkg.json</summary>

```
{
  "name": "lichtsteuerung",
  "version-string": "2.0.0",
  "dependencies": [
    "libfreenect2"
  ]
}

```
</details>

--- a/ports/libfreenect2/fix-macbuild.patch
+++ b/ports/libfreenect2/fix-macbuild.patch
@@ -1,0 +1,22 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index aedc576..4f16abc 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -165,6 +165,8 @@ SET(LIBFREENECT2_DLLS
+ SET(HAVE_VideoToolbox "no (Apple only)")
+ IF(APPLE)
+   FIND_LIBRARY(VIDEOTOOLBOX_LIBRARY VideoToolbox)
++  FIND_LIBRARY(CF_LIBRARY CoreFoundation)
++  FIND_LIBRARY(IOK_LIBRARY IOKit)
+ 
+   SET(HAVE_VideoToolbox no)
+   IF(VIDEOTOOLBOX_LIBRARY)
+@@ -184,6 +186,8 @@ IF(APPLE)
+       ${COREFOUNDATION_LIBRARY}
+       ${COREMEDIA_LIBRARY}
+       ${COREVIDEO_LIBRARY}
++      ${CF_LIBRARY}
++      ${IOK_LIBRARY}     
+     )
+   ENDIF(VIDEOTOOLBOX_LIBRARY)
+ ENDIF(APPLE)

--- a/ports/libfreenect2/fix-macbuild.patch
+++ b/ports/libfreenect2/fix-macbuild.patch
@@ -16,7 +16,7 @@ index aedc576..4f16abc 100644
        ${COREMEDIA_LIBRARY}
        ${COREVIDEO_LIBRARY}
 +      ${CF_LIBRARY}
-+      ${IOK_LIBRARY}     
++      ${IOK_LIBRARY}
      )
    ENDIF(VIDEOTOOLBOX_LIBRARY)
  ENDIF(APPLE)

--- a/ports/libfreenect2/portfile.cmake
+++ b/ports/libfreenect2/portfile.cmake
@@ -14,11 +14,6 @@ string(REPLACE "(WIN32)"
                "(WIN32_DISABLE)" FINDLIBUSB "${FINDLIBUSB}")
 file(WRITE "${SOURCE_PATH}/cmake_modules/FindLibUSB.cmake" "${FINDLIBUSB}")
 
-file(READ "${SOURCE_PATH}/examples/CMakeLists.txt" EXAMPLECMAKE)
-string(REPLACE "(WIN32)"
-               "(WIN32_DISABLE)" EXAMPLECMAKE "${EXAMPLECMAKE}")
-file(WRITE "${SOURCE_PATH}/examples/CMakeLists.txt" "${EXAMPLECMAKE}")
-
 vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
     opengl     ENABLE_OPENGL
     opencl     ENABLE_OPENCL
@@ -28,6 +23,7 @@ vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}"
     OPTIONS
         -DENABLE_CUDA=OFF
+        -DBUILD_EXAMPLES=OFF
         ${FEATURE_OPTIONS}
 )
 

--- a/ports/libfreenect2/portfile.cmake
+++ b/ports/libfreenect2/portfile.cmake
@@ -4,41 +4,41 @@ vcpkg_from_github(
     REF v0.2.0
     SHA512 3525e3f21462cecd3b198f64545786ffddc2cafdfd8146e5a46f0300b83f29f1ad0739618a07ab195c276149d7e2e909f7662e2d379a2880593cac75942b0666
     HEAD_REF master
-    PATCHES fix-dependency-libusb.patch
+    PATCHES
+        fix-dependency-libusb.patch
+        fix-macbuild.patch
 )
 
-file(READ ${SOURCE_PATH}/cmake_modules/FindLibUSB.cmake FINDLIBUSB)
+file(READ "${SOURCE_PATH}/cmake_modules/FindLibUSB.cmake" FINDLIBUSB)
 string(REPLACE "(WIN32)"
                "(WIN32_DISABLE)" FINDLIBUSB "${FINDLIBUSB}")
-file(WRITE ${SOURCE_PATH}/cmake_modules/FindLibUSB.cmake "${FINDLIBUSB}")
+file(WRITE "${SOURCE_PATH}/cmake_modules/FindLibUSB.cmake" "${FINDLIBUSB}")
 
-file(READ ${SOURCE_PATH}/examples/CMakeLists.txt EXAMPLECMAKE)
+file(READ "${SOURCE_PATH}/examples/CMakeLists.txt" EXAMPLECMAKE)
 string(REPLACE "(WIN32)"
                "(WIN32_DISABLE)" EXAMPLECMAKE "${EXAMPLECMAKE}")
-file(WRITE ${SOURCE_PATH}/examples/CMakeLists.txt "${EXAMPLECMAKE}")
+file(WRITE "${SOURCE_PATH}/examples/CMakeLists.txt" "${EXAMPLECMAKE}")
 
 vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
     opengl     ENABLE_OPENGL
     opencl     ENABLE_OPENCL
 )
 
-vcpkg_configure_cmake(
-    SOURCE_PATH ${SOURCE_PATH}
-    PREFER_NINJA
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
     OPTIONS
         -DENABLE_CUDA=OFF
-        # FEATURES
         ${FEATURE_OPTIONS}
 )
 
-vcpkg_install_cmake()
+vcpkg_cmake_install()
 
-vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/freenect2 TARGET_PATH share/freenect2)
+vcpkg_cmake_config_fixup(PACKAGE_NAME freenect2 CONFIG_PATH lib/cmake/freenect2)
 
 vcpkg_copy_pdbs()
 
-file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
 
-file(INSTALL ${SOURCE_PATH}/GPL2 DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
+file(INSTALL "${SOURCE_PATH}/GPL2" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
 
 vcpkg_fixup_pkgconfig()

--- a/ports/libfreenect2/vcpkg.json
+++ b/ports/libfreenect2/vcpkg.json
@@ -1,12 +1,21 @@
 {
   "name": "libfreenect2",
   "version-string": "0.2.0",
-  "port-version": 7,
+  "port-version": 8,
   "description": "Open source drivers for the Kinect for Windows v2 device",
   "homepage": "https://github.com/OpenKinect/libfreenect2",
+  "license": "GPL-2.0-only",
   "dependencies": [
     "libjpeg-turbo",
-    "libusb"
+    "libusb",
+    {
+      "name": "vcpkg-cmake",
+      "host": true
+    },
+    {
+      "name": "vcpkg-cmake-config",
+      "host": true
+    }
   ],
   "default-features": [
     "opengl"

--- a/ports/libfreenect2/vcpkg.json
+++ b/ports/libfreenect2/vcpkg.json
@@ -1,6 +1,6 @@
 {
   "name": "libfreenect2",
-  "version-string": "0.2.0",
+  "version": "0.2.0",
   "port-version": 8,
   "description": "Open source drivers for the Kinect for Windows v2 device",
   "homepage": "https://github.com/OpenKinect/libfreenect2",

--- a/scripts/ci.baseline.txt
+++ b/scripts/ci.baseline.txt
@@ -496,7 +496,6 @@ libevhtp:arm-uwp=fail
 libexif:arm-uwp=fail
 libexif:x64-uwp=fail
 libfreenect2:arm64-windows=fail
-libfreenect2:x64-osx=fail
 libgit2:arm-uwp=fail
 libgit2:x64-uwp=fail
 libgo:arm-uwp=fail

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -3558,7 +3558,7 @@
     },
     "libfreenect2": {
       "baseline": "0.2.0",
-      "port-version": 7
+      "port-version": 8
     },
     "libftdi": {
       "baseline": "0.20",

--- a/versions/l-/libfreenect2.json
+++ b/versions/l-/libfreenect2.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "0c6f0f09436ea7df652c15f4b886694bf0dbaae7",
+      "version": "0.2.0",
+      "port-version": 8
+    },
+    {
       "git-tree": "cdedefe776984f442e8af70b0e0738d12d56fafa",
       "version-string": "0.2.0",
       "port-version": 7

--- a/versions/l-/libfreenect2.json
+++ b/versions/l-/libfreenect2.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "0c6f0f09436ea7df652c15f4b886694bf0dbaae7",
+      "git-tree": "ce8aefbc76e982371258202989111294d9a5221d",
       "version": "0.2.0",
       "port-version": 8
     },

